.github/workflows/functional_tests.yaml

```diff
@@ -0,0 +1,89 @@
+on: issue_comment
+
+jobs:
+  pr_commented:
+    # This job only runs for pull request comments containing /functional
+    name: Functional Tests
+    if: contains(github.event.comment.body, '/test') && contains(github.event.comment.html_url, '/pull/')
```

Comment:
@tsebastiani the job gets triggered only if one of the project members in github comments /test right?